keyboards/hineybush/h87_g2/info.json

```
@@ -0,0 +1,118 @@
+{
+    "manufacturer": "Hiney LLC",
+    "keyboard_name": "h87_g2",
+    "maintainer": "hineybush",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B11", "B10", "B2", "B1", "B0", "A7", "A5", "A4", "A3", "F1", "C15", "C14", "F0", "C13", "B9", "B8", "B5"],
+        "rows": ["A15", "B3", "B4", "A0", "B6", "B7"]
+    },
+    "processor": "STM32F072",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0001",
+        "vid": "0x4048"
+    },
+    "layouts": {
+        "LAYOUT_all": {
```

This change will also need to be reflected in the keymaps.
```suggestion
        "LAYOUT": {
```